api/cf-rpc-types/src/lp.rs

```diff
@@ -135,3 +135,9 @@ impl TryFrom<CloseOrderJson> for CloseOrder {
 		})
 	}
 }
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct LiquidityDepositDetails {
```

Comment:
Technically details of a channel that will be deposited into, rather than a deposit itself
```suggestion
pub struct LiquidityDepositChannelDetails {
```